Makefile

```diff
@@ -44,6 +44,10 @@ VERSION ?= $(shell git describe --tags --always --dirty)
 # Set this to 1 to build a debugger-friendly binaries.
 DBG ?=
 
+# Username and password for manifest-tool.
```

Comment:
Small changes, please:

1) Rename to REGISTRY_USERNAME and REGISTRY_PASSWORD
2) Put next to REGISTRY above
3) Update comment like "Credentials to access the registry."